modules/sandboxed-containers-deleting-namespace-web-console.adoc

```diff
@@ -4,24 +4,24 @@
 
 :_content-type: PROCEDURE
 [id="sandboxed-containers-deleting-namespace-web-console_{context}"]
-= Deleting a namespace using the web console
+= Deleting {sandboxed-containers-first} namespace
 
-You can delete a namespace by using the {product-title} web console.
+After you run the preceding commands, your cluster is restored to the state that it was prior to the installation process. You can now revoke namespace access to the Operator by deleting the `openshift-sandboxed-containers-operator` namespace.
 
 .Prerequisites
 
-* You have {product-title} {product-version} installed on your cluster.
+* You have {product-title} 4.10 installed on your cluster.
```

Comment:
This looks counter-intuitive. If the idea is to list "4.10 or later", shouldn't we state that?

Reply:
Done: I changed this back across the files. It now gets updated automatically.

Reply:
Yes, my bad. I changed it back across the board.